csrankings-m.csv

```diff
@@ -1497,6 +1497,7 @@ Mian M. Awais,LUMS,https://lums.edu.pk/lums_employee/516,wkxlfFAAAAAJ
 Mian Muhammad Awais,LUMS,https://lums.edu.pk/lums_employee/516,wkxlfFAAAAAJ
 Miao Jin,University of Louisiana - Lafayette,http://www.ucs.louisiana.edu/~mxj9809,rhZtTHsAAAAJ
 Miao Qiao,Massey University,https://www.massey.ac.nz/massey/learning/colleges/college-of-sciences/staff-list.cfm?stref=689350,Guff-1wAAAAJ
+Miaohui Wang,Shenzhen University,https://charwill.github.io/,L_ptAR7tUAAAAJ

```

Comment:
correct scholar link appears to be L_wvjMsAAAAJ

Reply:
Thank you for your correction. I have changed the scholar link to the file.